pages.de/common/[[.md

```diff
@@ -0,0 +1,37 @@
+# [[
+
+> Teste und vergleiche Dateien und Variablen.
+> Gibt 0 zurück, wenn der Ausdruck wahr ist und 1 wenn nicht.
+> Weitere Informationen: <https://www.gnu.org/software/bash/manual/bash.html#index-_005b_005b>.
+
+- Überprüfe, ob eine bestimmte Variable gleich oder ungleich einem bestimmen String ist:
+
+`[[ ${{variable}} {{==|!=}} "{{string}}" ]]`
+
+- Überprüfe, ob ein bestimmter String einem bestimmten regulären Ausdruck oder glob-Muster entspricht:
+
+`[[ ${{variable}} {{==|=~}} {{muster}} ]]`
+
+- Überprüfe, ob eine Variable gleich, ungleich, größer, kleiner, größer-gleich oder kleiner-gleich als eine bestimme Zahl ist:
```

Comment:
```suggestion
- Überprüfe, ob eine Variable gleich/ungleich/größer/kleiner/größer-gleich oder kleiner-gleich als eine bestimme Zahl ist:
```

Reply:
Yep, I agree to both of your changes. As that's been copied from `[.md`, we'd need to change that too, though. What's the procedure for that? Should I create a new PR?

Reply:
I'd suggest you do.